timezone/src/include/timezone/dialogs.rb

```diff
@@ -572,22 +573,21 @@ def TimezoneDialog(args)
         # true by default (fate#303520)
         @ntp_used = true
         # configure NTP client
-        Builtins.srandom
-        @ntp_server = Builtins.sformat(
-          "%1.opensuse.pool.ntp.org",
-          Builtins.random(4)
-        )
+        # to prevent misusage of ntp.org we need to distinguish opensuse and SLE usage
+        base_products = Product.FindBaseProducts
+        if base_products.any? { |p| p["shortname"] == "openSUSE" }
```

Comment:
Here I'd prefer just `p["name"]` as that is the unique ID for the product, `shortname` or `displayname` are long texts which might contain anything displayable to the user, e.g. the `shortname`  might be changed to "openSUSE-16" or something like that in the future.

Maybe to make the test more robust we could make it less strict, e.g. `p["name"] =~ /openSUSE/i`